gnomad_qc/v3/annotations/generate_freq_data.py

```diff
@@ -142,9 +157,10 @@ def main(args):
             adj=get_adj_expr(mt.GT, mt.GQ, mt.DP, mt.AD),
         )
 
-        logger.info("Densify-ing...")
-        mt = hl.experimental.densify(mt)
-        mt = mt.filter_rows(hl.len(mt.alleles) > 1)
+        if not args.het_nonref_patch:
+            logger.info("Densify-ing...")
```

Comment:
```suggestion
            logger.info("Densifying...")
```
(there's no hyphen in the other script)